src/channel.cr

```diff
@@ -111,9 +117,11 @@ abstract class Channel(T)
         return ops.size, nil
       end
 
+      fiber = Fiber.current
+      fiber.callback = ->{ ops.each &.unwait(fiber); nil }
```

Comment:
You can drop `; nil`, since `#each` has `Nil` return type already.

Reply:
fixed

Reply:
It's still there :)

Reply:
oops...hopefully fixed this time :)

Reply:
It is indeed :)